WordPress/src/main/java/org/wordpress/android/ui/media/WordPressMediaUtils.java

```diff
@@ -280,6 +280,22 @@ public static Cursor getWordPressMediaVideos(String blogId) {
     }
 
     /**
+     * Given a VideoPress id, returns the corresponding remote video URL stored in the DB
+     * @param blogId the current blog id
+     * @param videoPressId the VideoPress id (used in VP shortcodes: [wpvideo the-VideoPress-id])
+     */
+    public static String getNetworkVideoUrlFromVideoPressId(String blogId, String videoPressId) {
+        Cursor cursor = WordPress.wpDB.getMediaFileByVideoPressId(blogId, videoPressId);
+
+        String path = "";
+        if (cursor != null && cursor.moveToFirst()) {
+            path = cursor.getString(cursor.getColumnIndex("fileURL"));
```

Comment:
Good point, I originally wrote that routine expecting I'd need to draw a thumbnail URL and possibly some meta-data from the media file also, and forgot to go back and adjust it. I'll limit it to just the fileURL and we can expand the fields in the future when we have thumbnail storage support.